sqlgen/reflect.go

```diff
@@ -369,6 +369,55 @@ func makeWhere(table *Table, filter Filter) (*SimpleWhere, error) {
 	}, nil
 }
 
+type BaseCountQuery struct {
```

Comment:
Do we have to export anything we added here?

Reply:
I don't believe so -- all of the other *basequeries are exported, but perhaps that's undesirable in hindsight.